docs/faq.md

```diff
@@ -0,0 +1,46 @@
+
+
+## How does Gnosis Protocol relate to Gnosis’ earlier DEX product, the DutchX trading protocol?
+Both the DutchX trading protocol and Gnosis Protocol come from a similar motivation: to implement batch auctions with uniform clearing prices. Gnosis Protocol keeps all the advantages of the DutchX trading protocol but adds: shorter trading times, allowing every participant to set a limit price (i.e. on the DutchX, sellers could only place market orders), and thanks to the ring trades, Gnosis Protocol does not split liquidity into separate trading pairs.
+
+
+## Will there be a version 2 of Gnosis Protocol?  
+Gnosis protocol has no admin key or other ability to upgrade, However, we will continue to innovate and suggest users to switch to a newer version if it can offer significant improvements over the previous one. A few areas of improvement could be: increased scalability and lower gas costs for usage, higher expressibility of orders, or the ability to submit sealed bids.
+
+
+## Is Gnosis Protocol secure?  
+The open source code on [Github](https://github.com/gnosis/dex-contracts). The verified contracts are on Etherscan for [Mainnet](https://etherscan.io/address/0x6f400810b62df8e13fded51be75ff5393eaa841f) and [Rinkeby](https://rinkeby.etherscan.io/address/0xC576eA7bd102F7E476368a5E98FA455d1Ea34dE2). The contracts have also carefully been audited by external smart contract security experts. You can find the audit report [here](https://github.com/gnosis/dex-contracts/raw/master/Exchange_audit_report.pdf).
+
+Since the end of January 2020, a Gnosis Protocol [bug bounty program](https://blog.gnosis.pm/2020-dex-bug-bounty-210f2b67a764) for up to $50,000 per bug report has been open.
+
+
+## What is Mesa?  
+[Mesa](https://mesa.eth.link/) is the first dapp built on the Gnosis Protocol. Mesa is a general trading interface that supports simple market making strategies for stablecoins. Mesa was launched by the [dxDAO](https://twitter.com/Dxdao_), an organization leading the evolution toward a more "de"-centralized DeFi.
+
+The dxDAO is a decentralized organization initialized in May of 2019, with over 400 unique stakeholder addresses. With a potential to grow its membership into the thousands, the community-owned and operated organization develops, governs, and promotes DeFi protocols. The dxDAO owns the DutchX trading protocol, and community members are working to launch a prediction market platform (Omen), a state of the art DEX (Mesa), a privacy-centric DeFi dashboard (Mix), and a fundraiser. 
+
+<a name="minimum-order"></a>
+## Why does the Mesa dapp have a warning regarding a minimum amount for orders?  
```

Comment:
`##` already creates an anchor tag,in this case it would be `docs/faq#why-does-the-mesa-dapp-have-a-warning-regarding-a-minimum-amount-for-orders`
What is `<a name="minimum-order"></a>` for? A shorter alias? `docs/faq#minimum-order`? Then please use `id` in place of `name`, `name` is deprecated for this purpose

Reply:
i would simply suggest the change @Velenir

Reply:
```suggestion
<a id="minimum-order"></a>
## Why does the Mesa dapp have a warning regarding a minimum amount for orders?  
```